influxdb.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/influxdb/influxdb/client"
 )
 
+const (
```

Comment:
Hmm, I think I had a "duh" moment there.  I thought I looked for constants but I clearly missed them.  Moved.